presto-docs/src/main/sphinx/functions/string.rst

```diff
@@ -14,7 +14,7 @@ String Functions
 
     These functions assume that the input strings contain valid UTF-8 encoded
     Unicode code points.  There are no explicit checks for valid UTF-8, and
-    the functions may return incorrect results or fail on invalid UTF-8.
```

Comment:
Are there no functions that fail?

This needs to remain if we keep the failure behavior of reverse.